### PR TITLE
fix: Replace unfriendly error messages

### PR DESCRIPTION
**Pull request by dd84ai**
Reviewed msgs that looked hurtful and not desirable And offered to replace with some alternative joke options

addressing https://github.com/python-discord/bot/issues/3152

Relevant discussion: https://discord.com/channels/267624335836053506/429409067623251969/1303867812854497303

### CODE REVIEW

File: bot/constants.py [hunk 1]
```diff
@@ -661,8 +661,9 @@ class _Keys(EnvConfig, env_prefix="api_keys_"):
     "In the future, don't do that.",
     "That was a mistake.",
     "You blew it.",
-    "You're bad at computers.",
-    "Are you trying to kill me?",
+    "Application bot.exe will be closed.",
+    "Kernel Panic! *Kernel runs around in panic*",
+    "error 418. I am a teapot.",
```
Comment (shenanigansd) [inline bot/constants.py:666]: nit: only one that doesn't start with a capital letter
```suggestion
    "Error 418. I am a teapot.",
```

**Review by shenanigansd — APPROVED** [review]
LGTM, quick win to improve friendliness

**Review by vivekashok1221 — APPROVED** [review]
LGTM

File: bot/constants.py [hunk 2]
```diff
@@ -627,7 +627,7 @@ class _Keys(EnvConfig, env_prefix="api_keys_"):
     "Not likely.",
     "No way, José.",
     "Not in a million years.",
-    "Fat chance.",
+    "I would love to, but unfortunately ... no.",
```
Comment (wookie184) [inline bot/constants.py:630]:
```suggestion
    "I would love to, but unfortunately... no.",
```

**Review by wookie184 — APPROVED** [review]
Looks good. I can merge when the suggested change I made is resolved.

`ERROR_REPLIES` doesn't actually seem to be used anywhere, so those changes wont do anything. [sir-lancebot has a copy of the same replies](https://github.com/python-discord/sir-lancebot/blob/1cf62552f77955c89fd9dd1631e0efa6391ad74c/bot/constants.py#L324) and it does use them, so it would be more useful to change them there.

We should move the lists into [bot-core](https://github.com/python-discord/bot-core) so we don't have this duplication, but that's a separate issue.

**Comment by dd84ai** [comment]
@wookie184 merged suggestions an opened duplicate in lancerbot https://github.com/python-discord/sir-lancebot/pull/1627

**Pull request merged**